src/metatrain/experimental/nanopet/model.py

```diff
@@ -136,7 +143,15 @@ def __init__(self, model_hypers: Dict, dataset_info: DatasetInfo) -> None:
         # time, and they are added to the output at evaluation time
         composition_model = CompositionModel(
             model_hypers={},
-            dataset_info=dataset_info,
+            dataset_info=DatasetInfo(
+                length_unit=dataset_info.length_unit,
+                atomic_types=self.atomic_types,
+                targets={
+                    target_name: target_info
+                    for target_name, target_info in dataset_info.targets.items()
+                    if is_valid_composition_model_target(target_info)
```

Comment:
should this be made into a staticmethod? So we remove one import and this becomes

```suggestion
                    if CompositionModel.is_valid_target(target_info)
```